i18n/ja/docusaurus-plugin-content-docs/current/guides/integration-options-private-operator.md

```diff
@@ -1,83 +1,108 @@
 ---
-title: Private Operator Integrations
-description: Overview of UID2 Private Operator options.
+title: UID2 Integrations for Private Operators
+description: Information summary for Private Operators.
 hide_table_of_contents: false
-sidebar_position: 02
 ---
 
 import Link from '@docusaurus/Link';
 
-# Private Operator Integrations
+# UID2 Private Operator Integration Overview
 
-Private Operator は、UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link> のプライベートインスタンスです。つまり、特定の組織が、自分自身の使用のために UID2 Operator のプライベートインスタンスを独占的にホストしています。
+Private Operators of UID2 send first-party <Link href="../ref-info/glossary-uid#gl-dii">directly identifying information (DII)</Link> to a secure environment for translation, and control the destinations for those identifiers. A participant that chooses to become a Private Operator can generate and manage UID2s, running a UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link> service in a private environment.
```

Comment:
Not sure if this is generated, but this should match my suggested changes above

Reply:
For now, it's a copy. Updated the copy. Thx.